helm/ecowitt2mqtt/values.yaml

```diff
@@ -0,0 +1,147 @@
+# Default values for ecowitt2mqtt.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: bachya/ecowitt2mqtt
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+mqtt:
+  auth:
+    enabled: false
+    username: username
+    password: password
+
+envvar:
+  - name: ECOWITT2MQTT_BATTERY_OVERRIDE
+    value: wh60_batt1=numeric
+    # a semicolon-delimited list of key=value battery overrides (default: numeric)
+  - name: ECOWITT2MQTT_CONFIG
```

Comment:
Since you're allowing this env var, note that it refers to a configuration specified via a JSON/YAML file. Does anything special need to be done to allow that to work in a multi-container/orchestrated environment?

Reply:
it would be needed to added to the chart as a configmap, to be honest though i would expect anyone that needs that to be capable of adding that to the chart and raising a PR. 

If you have an example file i can use i could create the configmap from that and add it to the chart. with an enable/disable var.

Reply:
Example config files are in the readme. I'll leave it to you whether you want to include it in this PR; your suggestion is great, too.

Reply:
Lets do it in another PR as I can also add an MQTT subchart at the same time to let people do an AIO deployment, but i'll do that with less urgency than this. :)